modular_newhorizons/master_files/code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -0,0 +1,24 @@
+/singleton/reagent/toxin/potassium_chlorophoride
+	description = "A specific chemical based on Potassium Chloride to stop the heart for surgery. Not safe to eat!"
+
+/singleton/reagent/lexorin
+	description = "Lexorin temporarily stops respiration. Causes tissue damage."
```

Comment:
"Lexorin is a complex toxin that temporarily stops respiration. It attempts to induce general hypoxia by weakening the diaphragm to prevent respiration and also by binding to haemoglobins to prevent oxygen molecules from doing the same."